docs/Users_Guide/glossary.rst

```diff
@@ -5456,15 +5457,33 @@ METplus Configuration Glossary
 
      | *Used by:*  EnsembleStat
 
+   GRID_STAT_MASK_GRID
+     Specify the value for 'mask.grid' in the MET configuration file for GridStat.
+
+     | *Used by:*  GridStat
+
+   GRID_STAT_MASK_POLY
+     Specify the value for 'mask.poly' in the MET configuration file for GridStat.
+
+     | *Used by:*  GridStat
+   
+   DESC
+     Specify the value for 'desc' in the MET configuration file for the MET tool being used
+
+    | *Used by:* GridStat, PointStat, EnsembleStat, GridDiag, MODE, MTD, SeriesAnalysis, TCGen, TCPairs, TCStat
+
    ENSEMBLE_STAT_MET_CONFIG_OVERRIDES
      Override any variables in the MET configuration file that are not
      supported by the wrapper. This should be set to the full variable name
      and value that you want to override, including the equal sign and the
      ending semi-colon. The value is directly appended to the end of the
      wrapped MET config file.
+     
      Example:
```

Comment:
A comment/request on this would be to provide 1 simple example (like desc = "override_desc") and 1 slightly more complex (i.e. output_flag = {eclv=BOTH;} ) example. Seeing how the bracketing works may help users teach themselves how to meet their own needs.

Reply:
@j-opatz there is a new section in the System Configuration page called "Overriding Unsupported MET config file settings" that dives into the details of settings these variables. The problem with the examples in the glossary is each tool has a different set of MET config variables, so the examples would have to be tailored to each tool to be accurate. I propose we remove these examples altogether since there is a link to the User's Guide section that describes how this works.